src/ds/d400/d400-factory.cpp

```diff
@@ -401,6 +401,48 @@ namespace librealsense
         };
     };
 
+    class rs421_device : public d400_active,
+                         public ds_advanced_mode_base,
+                         public firmware_logger_device
+    {
+    public:
+        rs421_device( std::shared_ptr< const d400_info > const & dev_info, bool register_device_notifications )
+            : device( dev_info, register_device_notifications )
+            , backend_device( dev_info, register_device_notifications )
+            , d400_device( dev_info )
+            , d400_active( dev_info )
+            , ds_advanced_mode_base( d400_device::_hw_monitor, get_depth_sensor() )
+            , firmware_logger_device(
+                  dev_info, d400_device::_hw_monitor, get_firmware_logs_command(), get_flash_logs_command() )
+        {
+            // Unregister unsupported options registered commonly by d400_device
+            auto & depth_sensor = get_depth_sensor();
+            depth_sensor.unregister_option( RS2_OPTION_OUTPUT_TRIGGER_ENABLED );
+            depth_sensor.unregister_option( RS2_OPTION_INTER_CAM_SYNC_MODE );
+        }
+
+        std::shared_ptr<matcher> create_matcher(const frame_holder& frame) const override;
+
+        std::vector<tagged_profile> get_profiles_tags() const override
+        {
+            std::vector<tagged_profile> tags;
+            auto usb_spec = get_usb_spec();
+            if (usb_spec >= platform::usb3_type || usb_spec == platform::usb_undefined)
```

Comment:
Should we support USB 2?

Reply:
Why not? Again, should be same to D430.
Also I know that FW support previous USB generations.